source/stylesheets/elements/_typography.scss

```diff
@@ -0,0 +1,46 @@
+html {
+  font-family: $tbds-font-sans-serif;
+  font-size: 100%;
+  line-height: 1.6;
+
+  @media (min-width: 38em) {
+    font-size: 106.25%;
```

Comment:
This is an interesting % amount, what does it translate to?

Reply:
In a browser whose base font size is `16px` (which is most, if not all, browsers), this would translate to `17px`.